tools/ci_build/github/linux/docker/inference/x64/python/cpu/scripts/install_centos.sh

```diff
@@ -4,4 +4,14 @@ set -e
 os_major_version=$(cat /etc/redhat-release | tr -dc '0-9.'|cut -d \. -f1)
 
 echo "installing for os major version : $os_major_version"
-yum install -y which gdb redhat-lsb-core expat-devel tar unzip zlib-devel make libunwind bzip2 bzip2-devel
+yum install -y which gdb redhat-lsb-core expat-devel tar unzip zlib-devel make libunwind bzip2 bzip2-devel perl-IPC-Cmd python3 openssl-devel wget
```

Comment:
Why do you need to install python3? The image already have all kinds of pythons under /opt . Installing an extra one could cause confusions.

Reply:
Removed.

Reply:
Then before you run this command, you should add one of the pythons to PATH

Reply:
For example, 
```
export PATH=/opt/python/cp38-cp38/bin/python3.8:$PATH
```

Reply:
replaced "export ...", now the package builds.